src/SortingAlgorithms.jl

```diff
@@ -560,4 +562,55 @@ function sort!(v::AbstractVector, lo::Int, hi::Int, ::TimSortAlg, o::Ordering)
     return v
 end
 
+
+## Intro sort
+
+function sort!(v::AbstractVector, lo::Int, hi::Int, a::IntroSortAlg, o::Ordering)
+    N = length(v)
+    if N > 1
```

Comment:
It's referring to the line below, and I think that's right.  Care to submit a pull request?